bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -70,10 +71,10 @@ def post_block(self, block: Block) -> HexStr:
             'block': block.__dict__
         })
 
-    def get_block_data(self, block_id: HexStr) -> Block:
+    def get_block_data(self, block_id: HexStr) -> BlockWrapper:
```

Comment:
Why is this called `get_block_data`?

Reply:
hmm, good question I didn't name it. Should I rename (to `get_block` I suppose)?

Reply:
I think so, yeah

Reply:
Done